tests/components/deconz/test_climate.py

```diff
@@ -668,6 +668,97 @@ async def test_climate_device_with_preset(hass, aioclient_mock):
         )
 
 
+async def test_climate_device_with_euronics(hass, aioclient_mock):
```

Comment:
This test needs to be updated to follow the structure as is implemented in this PR https://github.com/home-assistant/core/pull/47821/files.

1. Don't use deepcopy but rather patch.dict
2. Don't use gateway.api.event_handler but rather the new websocket fixture